PowershellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingService.cs

```diff
@@ -506,7 +607,18 @@ public string GetPrompt()
             {
                 _currentPowerShell.Runspace = _runspace;
                 _currentPowerShell.AddCommand("prompt");
-                return _currentPowerShell.Invoke<string>().FirstOrDefault();
+
+                string prompt;
+                if (_runspace.ConnectionInfo == null)
+                {
+                    prompt = _currentPowerShell.Invoke<string>().FirstOrDefault();

```

Comment:
Do we want to pull the duplicated _currentPowerShell.Invoke<string>().FirstOrDefault(); up out of the if/else block?